docs/en_US/Compression/QuickStart.rst

```diff
@@ -0,0 +1,212 @@
+to the weight parameter of modules. 'input' means applying quantization operation to the input of module forward method. 'output' means applying quantization operation to the output of module forward method, which is often called as 'activation' in some papers.
+
+
+* **quant_bits** : int or dict of {str : int}
+
+bits length of quantization, key is the quantization type, value is the quantization bits length, eg. 
+
+.. code-block:::: bash
+
+   {
+       quant_bits: {
+           'weight': 8,
+           'output': 4,
+           },
+   }
+
+when the value is int type, all quantization types share same bits length. eg. 
+
+.. code-block:::: bash
+
+   {
+       quant_bits: 8, # weight or output quantization are all 8 bits
+   }
+
+The following example shows a more complete ``config_list``\ , it uses ``op_names`` (or ``op_types``\ ) to specify the target layers along with the quantization bits for those layers.
+
+.. code-block:::: bash
+
+   configure_list = [{
+           'quant_types': ['weight'],        
+           'quant_bits': 8, 
+           'op_names': ['conv1']
+       }, {
+           'quant_types': ['weight'],
+           'quant_bits': 4,
+           'quant_start_step': 0,
+           'op_names': ['conv2']
+       }, {
+           'quant_types': ['weight'],
+           'quant_bits': 3,
+           'op_names': ['fc1']
+           },
+          {
+           'quant_types': ['weight'],
+           'quant_bits': 2,
+           'op_names': ['fc2']
+           }
+   ]
+
+In this example, 'op_names' is the name of layer and four layers will be quantized to different quant_bits.
+
+APIs for Updating Fine Tuning Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some compression algorithms use epochs to control the progress of compression (e.g. `AGP <https://nni.readthedocs.io/en/latest/Compression/Pruner.html#agp-pruner>`__\ ), and some algorithms need to do something after every minibatch. Therefore, we provide another two APIs for users to invoke: ``pruner.update_epoch(epoch)`` and ``pruner.step()``.
+
+``update_epoch`` should be invoked in every epoch, while ``step`` should be invoked after each minibatch. Note that most algorithms do not require calling the two APIs. Please refer to each algorithm's document for details. For the algorithms that do not need them, calling them is allowed but has no effect.
+
+Export Compressed Model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can easily export the compressed model using the following API if you are pruning your model, ``state_dict`` of the sparse model weights will be stored in ``model.pth``\ , which can be loaded by ``torch.load('model.pth')``. In this exported ``model.pth``\ , the masked weights are zero.
+
+.. code-block:::: bash
+
+   pruner.export_model(model_path='model.pth')
+
+``mask_dict`` and pruned model in ``onnx`` format(\ ``input_shape`` need to be specified) can also be exported like this:
```

Comment:
Backslash?

Reply:
Blackslash is for whitespace. It's converted from markdown. Maybe we can get rid of this later.